src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -63,20 +63,24 @@ class Artwork extends React.Component<RelayProps, any> {
 
   saleMessage() {
     const artwork = this.props.artwork
-    if (artwork.is_in_auction && artwork.sale_artwork.sale.is_open) {
-      const numberOfBids = artwork.sale_artwork.bidder_positions_count
-      let text = artwork.sale_artwork.opening_bid.display
-      if (numberOfBids > 0) {
-        text = `${artwork.sale_artwork.current_bid.display} (${numberOfBids} bid${numberOfBids === 1 ? "" : "s"})`
+    if (artwork.is_in_auction) {
+      if (artwork.sale_artwork.sale.is_open) {
+        const numberOfBids = artwork.sale_artwork.bidder_positions_count
+        let text = artwork.sale_artwork.opening_bid.display
+        if (numberOfBids > 0) {
+          text = `${artwork.sale_artwork.current_bid.display} (${numberOfBids} bid${numberOfBids === 1 ? "" : "s"})`
```

Comment:
It _could_ but eigen would still need to compute it manually (the part in parens is in a different font colour).